### DISCUSSION TITLE
Request for bridged network adaptor

### DISCUSSION THREAD

**Discussion by CodeMonkeyCybersecurity** [utmapp/UTM#6231] [UTM Help (macOS)]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
Unable to connect machine to my local subnet using the shared network adaptor.
Need a bridged network adaptor please!
local subnet is 192.168.20.0/24
Subnet for VMs in 192.168.64.0/24
I have tried everything I know and can still not get it to talk with my local VLAN. Internet connectivity is fine

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: Sonoma 14.3.1
* Mac Chip (Intel, M1, ...): M2 Mac Mini and 2018 Intel Mac Mini

**Crash log**  
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either UTM, QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**  
For all issues related to running a VM, _including_ crashes, you should attach a debug log. (This is unavailable for macOS-on-macOS VMs. Attach an excerpt of your system log instead.)
To get the Debug log: open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**  
If your issue is related to a specific VM, please upload the config.plist from your VM's .utm directory. To get this, right-click the VM in UTM and choose "Show in Finder". Then right-click the selected file in Finder and choose "Show Package Contents". The config.plist file is now visible. Right-click it and choose "Compress". Attach the resulting config.plist.zip file here.
You can upload the entire .utm if needed, but note this includes your VM's drive image and may contain personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by adeadman** [comment]
This is all in the documentation.
[Apple Virtualisation](https://docs.getutm.app/settings-apple/devices/network/)
[QEMU](https://docs.getutm.app/settings-qemu/devices/network/network/)

Also see #4085 

It's worth searching either issues or the docs page first before opening an issue, as the bold sentence in the top line says. If the docs or existing issues don't already cover what you need, then you could open a new one with the detail of what is different about your request.
